soh/src/code/z_message_PAL.c

```diff
@@ -3384,7 +3384,7 @@ void Message_Update(PlayState* play) {
             }
             sLastPlayedSong = 0xFF;
             osSyncPrintf("OCARINA_MODE=%d   chk_ocarina_no=%d\n", play->msgCtx.ocarinaMode, msgCtx->unk_E3F2);
-            CheckTracker_OnMessageClose();
+            // TODO: OnMessageClose hook
```

Comment:
What's the context here? Could we get it clarified in the comment what this is going to be for?